packages/core/src/components/MediaPlayer/AudioPlayer/HeadlessAudioPlayer.tsx

```diff
@@ -97,6 +101,7 @@ const HeadlessAudioPlayer = React.forwardRef<
 
       const { sound } = await Audio.Sound.createAsync(finalSource);
       setCurrentSound(sound);
+      sound.setIsLoopingAsync(isLooping);
```

Comment:
This will only set the value when the audio is reloaded (audio source changes). Maybe we should have a separate useEffect that updates the looping state

```tsx
React.useEffect(() => {
    currentSound?.setIsLoopingAsync(isLooping);
}, [isLooping])
```

Reply:
We can remove this one now since we have the other useEffect @sieu-db 

```suggestion
```

Reply:
Otherwise all good!

Reply:
Yep, got it, thanks